This test fails if tried with python2 since it builds a 64-bit object and
then tries to run it with the 32-bit version of python2.

For now, just disable python2 from the test matrix so the python3 version
can be correctly tested.

diff --git a/test cases/python/8 different python versions/test.json b/test cases/python/8 different python versions/test.json
index fe75a1c26..5f88cc716 100644
--- a/test cases/python/8 different python versions/test.json	
+++ b/test cases/python/8 different python versions/test.json	
@@ -3,7 +3,6 @@
     "options": {
       "python": [
         { "val": null },
-        { "val": "python2" },
         { "val": "python3" },
         { "val": "pypy" },
         { "val": "pypy3" }
